Source/WebGPU/WebGPU/RenderPassEncoder.mm

```diff
@@ -450,8 +452,9 @@ static void setViewportMinMaxDepthIntoBuffer(auto& fragmentDynamicOffsets, float
 
 void RenderPassEncoder::incrementDrawCount(uint32_t drawCalls)
 {
-    m_drawCount += drawCalls;
-    if (m_drawCount > m_maxDrawCount)
+    auto checkedDrawCount = checkedSum<uint64_t>(m_drawCount, drawCalls);
+    m_drawCount = checkedDrawCount;
```

Comment:
if checkedDrawCount overflows, I think this will crash because m_drawCount is a uint64_t and the definition of operator T in CheckedArithmetic.h is:
```
    operator T() const
    {
        if (UNLIKELY(this->hasOverflowed()))
            this->crash();
        return m_value;
    }
```

Reply:
Proposed change, from:
```
m_drawCount = checkedDrawCount;
if (checkedDrawCount.hasOverflowed() || checkedDrawCount > m_maxDrawCount)
```
to:
```
m_drawCount = checkedDrawCount.hasOverflowed() ? (m_maxDrawCount + 1) : checkedDrawCount.value();
if (m_drawCount > m_maxDrawCount)
```

Reply:
technically the +1 would overflow resulting in no limit if someone uses 2^64 - 1 for maxDrawCount but they will be waiting a long time for 2^64 -1 draw commands to occur, so I don't think this is an issue in practice